Cargo.toml

```diff
@@ -20,10 +20,11 @@ pomelo = { version = "0.1.4", optional = true }
 thiserror = "1.0.21"
 serde = { version = "1.0", features = ["derive"], optional = true }
 petgraph = { version ="0.5", optional = true }
+pp-rs = { git = "https://github.com/Kangz/glslpp-rs", branch = "main", optional = true }
```

Comment:
we need to specify the exact revision instead of the branch. Since we don't vendor a lockfile, the branch can move.

Reply:
would this work for now, or do we need it on crates.io?

Reply:
we don't need it on crates until we need to publish Naga-0.4

Reply:
added specific rev, resolving for now